packages/@aws-cdk/aws-ecs-patterns/test/fargate/test.load-balanced-fargate-service-v2.ts

```diff
@@ -386,6 +386,73 @@ export = {
 
       test.done();
     },
+
+    'test with enableExecuteCommand set to true'(test: Test) {
+      // GIVEN
+      const stack = new Stack();
+      const vpc = new Vpc(stack, 'VPC');
+      const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+
+      // WHEN
+      new ApplicationLoadBalancedFargateService(stack, 'Service', {
+        cluster,
+        taskImageOptions: {
+          image: ecs.ContainerImage.fromRegistry('test'),
+        },
+        enableExecuteCommand: true,
+      });
+
+      // THEN - stack contains a service with execute-command enabled
+      expect(stack).to(haveResourceLike('AWS::ECS::Service', {
+        EnableExecuteCommand: true,
+      }));
+
+      test.done();
+    },
+
+    'test with enableExecuteCommand omitted'(test: Test) {
+      // GIVEN
+      const stack = new Stack();
+      const vpc = new Vpc(stack, 'VPC');
+      const cluster = new ecs.Cluster(stack, 'Cluster', { vpc });
+
+      // WHEN
+      new ApplicationLoadBalancedFargateService(stack, 'Service', {
+        cluster,
+        taskImageOptions: {
+          image: ecs.ContainerImage.fromRegistry('test'),
+        },
+      });
+
+      // THEN - stack contains a service with execute-command enabled
+      expect(stack).to(haveResourceLike('AWS::ECS::Service', {
+        EnableExecuteCommand: ABSENT,
```

Comment:
Does default value expected is ABSENT?  or we do we expect false as a default value?

Reply:
Hey sorry for the late response. As implemented the default for the interface is to leave that property undefined as an optional with `readonly enableExecuteCommand?: boolean;`

If you're arguing that we shouldn't be allowed to omit that property and force a true/false then I can make those changes.

Unrelated: I'll fix the copy/pasta comments that are incorrect.

Reply:
I don't think we need either of the omitted or false tests. The `ABSENT` test asserts whether the prop is `undefined` or not set (a check not necessary in our case) and the `false` test below essentially checks if the `enableExecuteCommand` is set correctly and so does the `true` test. It would be great if you could remove both these unit tests.

Reply:
Hey, thanks for the review!

> I don't think we need either of the omitted or false tests. The ABSENT test asserts whether the prop is undefined or not set (a check not necessary in our case)

Why wouldn't it be necessary to test `ABSENT` behavior? It's a valid config and what most (all) people will be using prior to this PR being merged. It's a good check of back compatibility and that the underlying templates that get synthed aren't changing for people that omit this option.

> and the false test below essentially checks if the enableExecuteCommand is set correctly and so does the true test. It would be great if you could remove both these unit tests.

I'd argue that we should be testing to ensure that the proper value is getting set all of the way through for each possible input and not just the happy (true) path. These act as regression tests for each of possible configs related to this feature. If the guts of the implementation change and break things then these tests will fail.